sdk/keyvault/azure-security-keyvault/tests/test_examples_keys.py

```diff
@@ -0,0 +1,372 @@
+        created_key = key_client.create_key("keyrec", "RSA")
+        key_name = created_key.name
+        try:
+            # [START backup_key]
+            # backup key
+            key_backup = key_client.backup_key(key_name)
+
+            # returns the raw bytes of the backed up key
+            print(key_backup)
+
+            # [END backup_key]
+
+            key_client.delete_key("keyrec")
+            if self.is_live:
+                # wait a second to ensure the key has been deleted
+                time.sleep(20)
+            # [START restore_key]
+
+            # restores a backed up key
+            restored_key = key_client.restore_key(key_backup)
+            print(restored_key.id)
+            print(restored_key.version)
+
+            # [END restore_key]
+        except ResourceExistsError:
+            pass
+
+    @ResourceGroupPreparer()
+    @VaultClientPreparer(enable_soft_delete=True)
+    def test_example_keys_recover_purge(self, vault_client, **kwargs):
+        key_client = vault_client.keys
+        created_key = key_client.create_key("key-name", "RSA")
+        key_client.delete_key(created_key.name)
+        if self.is_live:
+            # wait a second to ensure the key has been deleted
+            time.sleep(30)
+
+        try:
+            # [START get_deleted_key]
+            # gets a deleted key (requires soft-delete enabled for the vault)
+            deleted_key = key_client.get_deleted_key("key-name")
+            print(deleted_key.name)
+
+            # [END get_deleted_key]
+        except ResourceNotFoundError:
+            pass
+
+        try:
+            # [START recover_deleted_key]
+
+            # recover deleted key to its latest version
+            recover_deleted_key = key_client.recover_deleted_key("key-name")
+            print(recover_deleted_key.id)
+            print(recover_deleted_key.name)
+
+            # [END recover_deleted_key]
+        except HttpResponseError:
+            pass
+
+        try:
+            key_client.delete_key("key-name")
+            if self.is_live:
+                # wait a second to ensure the key has been deleted
+                time.sleep(20)
+            # [START purge_deleted_key]
+
+            # if the vault has soft-delete enabled, purge permanently deletes the key
+            # (without soft-delete, an ordinary delete is permanent)
+            # key must be deleted prior to be purged
+            key_client.purge_deleted_key("key-name")
+
+            # [END purge_deleted_key]
+        except HttpResponseError:
+            pass
+
+    @ResourceGroupPreparer()
+    @VaultClientPreparer()
+    def test_key_wrap_and_unwrap(self, vault_client, **kwargs):
+        self.assertIsNotNone(vault_client)
+        client = vault_client.keys
+        key_name = self.get_resource_name("keywrap")
+
+        # create key
+        created_key = client.create_key(key_name, "RSA")
+        self.assertIsNotNone(created_key)
+
+        # [START wrap_key]
```

Comment:
I'm dubious on the value of these un/wrap snippets without more context around the operation and its parameters. It's okay to leave them out of the examples for now.